src/host/main.cpp

```diff
@@ -743,8 +743,7 @@ int main(int argc, char** argv)
       else
       {
         LOG_FAIL_FMT(
-          "No snapshot found. Node will request transactions all historical "
-          "transactions");
+          "No snapshot found. Node will request all historical transactions.");
```

Comment:
```suggestion
          "No snapshot found. Node will request all historical transactions");
```

As discussed this morning, I think we decided that our log lines don't end in periods? We could follow libfmt's lead and also try to enforce this at compile-time...